deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,48 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+nav{
+    flex-basis: 150px;
```

Comment:
Op het voorbeeld lijkt de Nav in verhouding wat breder meer richting de 300px of 30% ofzo. Is maar een detail.